etna/models/nn/deepar_new/sampler.py

```diff
@@ -0,0 +1,26 @@
+from etna import SETTINGS
```

Comment:
Looks like we should refactor base classes to make it possible to pass sampler in proper way, lets remove this logic for now